src/components/safe-messages/MsgModal/index.tsx

```diff
@@ -151,7 +151,7 @@ const MsgModal = ({
             Message:
```

Comment:
It should also be possible to copy the entire message here as well, like we do in the transaction list.